README.md

```diff
@@ -104,16 +112,23 @@ following are available:
    should contain an object where the property names are the variable names to
    use and the values are the constant values to use.
 
-### Lookup Sources
-Lookups are functions or tables available to olives. For instance, suppose a
-BED file is needed for different projects. It is convenient to turn this into a
-lookup. The following are available:
-
+### Function Sources
+Function sources provide functions (or lookup tables) available to olives. For
+instance, suppose a BED file is needed for different projects. It is convenient
+to turn this into a function. The following are available:
+
+- Built-in. There are built-in functions for common data manipulation
+- JIRA. This provides commands to count tickets matching certain words and to
+  retrieve ticket summaries and descriptions.
+- SFTP (in `lookup-sftp` directory). Create a JSON file ending in `.sftp`
+  containing `{"host":..., "port":22, "user":...}` to access file metadata via SFTP.
+- RunScanner. Create a JSON file ending in `.runscanner` containing
+  `{"url":...}` to access run information from [MISO RunScanner]http://github.com/TGAC/miso-lims().
```

Comment:
I think the MISO url needs to be surrounded by those parens

Reply:
Yeah. I'm losing it.